### ISSUE TITLE
Selecting files based which satisfy multiple criteria.

### ISSUE THREAD

**Issue by rtevans01** [qarmin/czkawka#295]
Is there a way to select duplicate files which match at least 2 conditions? For example, I want to select the newest files but those newest files must have the shortest directory in a duplicate group. Would this be possible? It would make the targeting process much more precise. Thanks.

**Comment by gessel** [comment]
Yes please, this is very useful in dupeGuru

**Comment by poisonborz** [comment]
Yes, just wanted to add this issue as well... On the gui, it would be awesome if instead of tabs there were checkboxes, so multiple could be selected at once. In the file list, there could be a "category" column to show what kind of error it is (duplicate, zeroed, etc).

**Comment by qarmin** [comment]
It is already added to Krokiet
